tests/integrations/django/asgi/test_asgi.py

```diff
@@ -1,16 +1,24 @@
 import pytest
 
+import django
 
 from channels.testing import HttpCommunicator
 
 from sentry_sdk import capture_message
 from sentry_sdk.integrations.django import DjangoIntegration
 
-from tests.integrations.django.myapp.asgi import application
+from tests.integrations.django.myapp.asgi import channels_application
 
+APPS = [channels_application]
+if django.VERSION >= (3, 0):
+    from tests.integrations.django.myapp.asgi import asgi_application
 
+    APPS += [asgi_application]
+
+
+@pytest.mark.parametrize("application", APPS)
```

Comment:
Why add this now when there is only one (unused) parameter?

Reply:
The `application` used to be imported from `tests.integration.django.myapp.asgi` at the top of this file and is now a pytest parameter. It is used in line 25 of this file which didn't change and is therefore outside of this diff.

`APPS` contains all the ASGI apps we want to test. If we are running with Django 2 installed, it will contain only one item, but if Django 3 is installed, it will contain both `asgi_application` and `channels_application`.

In `tox.ini` we install a lot of different Django versions and here we try to figure out what we can actually test under that Django version.